jerry-core/ecma/operations/ecma-atomics-object.c

```diff
@@ -29,6 +29,7 @@
 
 #include "jcontext.h"
 #include "jmem.h"
+#include "opcodes.h"
```

Comment:
opcodes-ecma-{arithmetics, bitwise} isn't enough here?

Reply:
Both .c files use this header.